lib/src/logic/validators.dart

```diff
@@ -264,7 +264,7 @@ class ShouldInBetweenDatesValidator<KeyType>
 
   @override
   bool isValid(DateTime date) {
-    return (date.compareTo(min) > 0 && date.compareTo(max) < 0);
+    return (date.isBetween(min, max));
```

Comment:
We are changing the behavior of the validator here, which might break already dependent code. Let's cover both behaviors here. We could add a new field to be passed in constructor, called "inclusive" with default value of false. When this flag is false the old behavior is maintained, otherwise the new behavior is used. We should also pass it as true for our specifc case in the `fazua_toolbox`